andy_li/Python Fundamentals/Type List.py

```diff
@@ -0,0 +1,32 @@
+def type_lst(lst):
+    my_str = ""
+    my_int = 0
+
+    for val in lst:
+        val_type = type(val)
+
+        if val_type is str:
+            if val is lst[len(lst) - 1]:
+                my_str += val
+            else:
+                my_str += (val + " ")
+
+        if val_type is int or val_type is float:
+            my_int += val
+

```

Comment:
The following blocks have some code duplication. Remember in programming that we always try to implement "DRY" practices. The previous lines were done very well. Great job